modules/math/src/main/java/com/opengamma/strata/math/impl/interpolation/Interpolator1D.java

```diff
@@ -20,30 +20,26 @@
  * A base class for interpolation in one dimension.
  */
 public abstract class Interpolator1D
-    implements Interpolator<Interpolator1DDataBundle, Double>, Serializable {
+    implements Serializable {
```

Comment:
This PR leaves `Serializable` unchanged from before. I'll take a bigger look in the next commit.